### PR TITLE
Add & fix content platform forms roles and namespaces

### PR DESCRIPTION
**Pull request by MatthewCane**
# Description

Fix the dev namespace name and add the prod namespace and role

# Testing

# Rollout procedure

Normal CDK deploy

# Breaking changes

None

# Checklist

- [ ] Tested on test cluster
- [ ] Added / updated automated tests (if necessary)
- [ ] Updated the documentation (if necessary)

### CODE REVIEW

File: config/clusters/cab_prod/resources.py [hunk 1]
```diff
@@ -276,3 +276,19 @@ def __init__(self, scope: Construct, id: str, platform_rbac: PlatformRbac, **kwa
         platform_rbac.grant_manage_namespace_resources(
             self, prod_witnessbox_v2_ns, witnessbox_v2_cdk_user
         )
+
+        content_platform_forms = UserNamespace(
```
Comment (ruthwells) [inline config/clusters/cab_prod/resources.py:280]: do you want to use names that end in `_ns` for namespaces and `_user` for users, as we've done for wb2? Makes things a bit more intuitive?

Comment (MatthewCane) [reply config/clusters/cab_prod/resources.py]: Updated

File: config/clusters/cab_prod/resources.py [hunk 2]
```diff
@@ -276,3 +276,19 @@ def __init__(self, scope: Construct, id: str, platform_rbac: PlatformRbac, **kwa
         platform_rbac.grant_manage_namespace_resources(
             self, prod_witnessbox_v2_ns, witnessbox_v2_cdk_user
         )
+
+        content_platform_forms_ns = UserNamespace(
+            self, "content-platform-forms", name="prod-content-platform-forms", isolated=False
```
Comment (ruthwells) [inline config/clusters/cab_prod/resources.py:281]: also (sorry, didn't notice it before) do you want to specify a slack channel, or is that not relevant in this case?

Comment (MatthewCane) [reply config/clusters/cab_prod/resources.py:281]: That's not relevant at the moment, I'll add one if required.

**Review by ruthwells — APPROVED** [review]
looks reasonable to me!

**Pull request merged**